internal/gok/overwrite.go

```diff
@@ -70,6 +70,11 @@ func (r *overwriteImplConfig) run(ctx context.Context, args []string, stdout, st
 		return err
 	}
 
+	sbom, _, err := packer.GenerateSBOM(cfg)
```

Comment:
Here and in the other GenerateSBOM() call, can you please add a comment to the tune of:

// GenerateSBOM() must be called before any modifications to cfg.InternalCompatibilityFlags, as the SBOM should reflect what’s going into gokrazy, not its internal implementation details.

Reply:
Sure :) done